### PR TITLE
Fix the serialization problem with current JSON of notification model

### LINKED ISSUE #73: Notification item not serialized
When serializing the notification (NotificationRequest) the property NotificationItemContainers (a list of NotificaticationItemContainer) comes null (no Json property assigned)

### PR DESCRIPTION
**Pull request by guibranco**
Fix #73 add JsonProperty attribute on property NotificationRequest.NotificationItemContainers

### CODE REVIEW

**Comment by AlexandrosMor** [comment]
Thank you for the PR @guibranco. Is it possible for you to add a unit test around this issue?

**Comment by guibranco** [comment]
- Added tests for the notifications (Uncomment test template in  **Adyen.EcommLibrary.Test/NotificationTest.cs**
- Need to change the project to add the **Mocks/*.json** files to output dir.
- Add **JsonPropertyAttribute** to **NotificationRequest** and **NotificationRequestItemContainer.cs** classes
- Fix **NotificationHandler.cs** behavior

File: Adyen.EcommLibrary.Test/Adyen.EcommLibrary.Test.csproj [hunk 1]
```diff
@@ -23,11 +23,8 @@
```
Comment (AlexandrosMor) [inline Adyen.EcommLibrary.Test/Adyen.EcommLibrary.Test.csproj]: Please do not commit any project file

**Comment by AlexandrosMor** [comment]
Hello @guibranco,
The project file Adyen.EcommLibrary.Test.csproj is now deleted. Please restore the file but not commit changes in csproj if it is not needed. 

Thank you in advance,
Alexandros Moraitis
Adyen

**Comment by guibranco** [comment]
Done!

File: Adyen.EcommLibrary.Test/NotificationTest.cs [hunk 2]
```diff
@@ -1,52 +1,102 @@
...
-
-        //[TestMethod]
-        //public void TestCaptureSuccess()
-        //{
-        //    var client = base.CreateMockTestClientRequest("Mocks/Notification/capture-true.json");
-        //}
+        [TestMethod]
+        public void TestAuthorisationSuccess()
+        {
+            //var jsonRequest = base.MockFileToString("Mocks/Notification/authorisation-true.json");
```
Comment (AlexandrosMor) [inline Adyen.EcommLibrary.Test/NotificationTest.cs]: var mockPath = GetMockFilePath("Mocks/notification/authorisation-true.json");
 var jsonRequest = MockFileToString(mockPath);
  I would recommend  something like this

**Comment by dietermijle** [comment]
+1

File: Adyen.EcommLibrary.Test/NotificationTest.cs (hunk 2 above)
Comment (guibranco) [reply Adyen.EcommLibrary.Test/NotificationTest.cs]: Done!

**Comment by guibranco** [comment]
Any news ? We are waiting the release of the package to deploy our integration...

**Comment by AlexandrosMor** [comment]
Fixes #78

**Pull request merged**